README.md

```diff
@@ -2,4 +2,97 @@
 
 # Awesome FSRS [![Awesome](https://cdn.rawgit.com/sindresorhus/awesome/d7305f38d29fed78fa85652e3a63e154dd8e8829/media/badge.svg)](https://github.com/sindresorhus/awesome)
 
- A curated list of awesome FSRS implementations, papers and resources. Feel free to suggest new projects in Issues or PR directly.
+A curated list of awesome FSRS implementations, papers and resources. Feel free to suggest new projects in Issues or PR directly.
+
+## Implementation
+
+- Python
+  - Scheduler (v5): [py-fsrs](https://github.com/open-spaced-repetition/py-fsrs)
+  - Optimizer: [fsrs-optimizer](https://github.com/open-spaced-repetition/fsrs-optimizer)
+  - [Deprecated] Optimizer: [fsrs-optimizer-tiny](https://github.com/open-spaced-repetition/fsrs-optimizer-tiny)
+- Rust
+  - Scheduler (v5) + Optimizer: [fsrs-rs](https://github.com/open-spaced-repetition/fsrs-rs)
+    - Run in browsers: [fsrs-browser](https://github.com/open-spaced-repetition/fsrs-browser)
+- TypeScript
+  - Scheduler (v5): [ts-fsrs](https://github.com/open-spaced-repetition/ts-fsrs)
+- Go
+  - Scheduler (v5): [go-fsrs](https://github.com/open-spaced-repetition/go-fsrs)
+- JavaScript
+  - Scheduler (v4): [fsrs.js](https://github.com/open-spaced-repetition/fsrs.js)
```

Comment:
Currently, there are no plans to continue maintaining `fsrs.js`.

Reply:
I suggest removing fsrs.js from the list.

Reply:
I do feel like I want to keep it listed the same way I listed `fsrs-optimizer-tiny` with `[Deprecated]` sign. But I followed the suggestion regardless, since I also notice there aren't much anything going on with the library.
Perhaps we can get the repository archived to discourage it?